### ISSUE TITLE
Multiple Files does not work: Error|RuntimeMethodInfo.UnsafeInvokeInternal 

### ISSUE THREAD

**Issue by runecalico** [ShokoAnime/ShokoServer#774]
﻿## PLEASE READ BEFORE POSTING YOUR ISSUE

Please create GitHub issues only for bugs and feature requests. If you need help with using Shoko or have a question, join us on Discord.

https://discord.gg/vpeHDsg

===== Guidelines ====

- Make sure you are using the latest **Stable** or **Daily** version of Shoko Server otherwise your issue will be closed.
- Search this repository (top of the page) for the issue or feature you are reporting, make sure it has not been fixed or reported before.
- If reporting an issue, then you must attach the relevant Shoko Server and Shoko Desktop **LOG FILE**. 
- If requesting a new feature, please explain in detail the feature itself and the benefits of it being added.
- If requesting cosmetic changes, please explain in detail why they should be made. 
- The template below **MUST BE USED** when submitting an issue, no exceptions.

## VERSION INFORMATION

Server Version: 3.9.4.30902

Desktop Version: 3.9.4.0

If you've compiled your own version, please use the last commit you compiled.

## LOG FILE

[2019-02-07.txt](https://github.com/ShokoAnime/ShokoServer/files/2843345/2019-02-07.txt)

## DESCRIPTION

Attempts to use the Multiple Files feature in Shoko Desktop immediately fail.

Logs on the server show this error:
[2019-02-07 19:55:12:605] Error|RuntimeMethodInfo.UnsafeInvokeInternal => RuntimeMethodHandle.InvokeMethod => ShokoServiceImplementation.GetAllEpisodesWithMultipleFiles System.NullReferenceException: Object reference not set to an instance of an object.
   at Shoko.Server.ShokoServiceImplementation.GetAllEpisodesWithMultipleFiles(Int32 userID, Boolean onlyFinishedSeries, Boolean ignoreVariations) in D:\Documents\GitHub\ShokoServer\Shoko.Server\API\v1\Implementations\ShokoServiceImplementation\ShokoServiceImplementation_Utilities.cs:line 1252

## STEPS TO REPRODUCE

Go to Shoko Desktop Client
Utilities tab
Multiple Files Tab
Hit Refresh

Please advise.

Unfortunately it seems this is an relatively unavoidable issue in my setup, as I've been working on getting  everything into Shoko for the last month and this seems to happen every time I blow everything away and start over.... So I figured I'd open a ticket and see if anyone can point me to where in the SQLite DB it might be barfing.

**Comment by da3dsoul** [comment]
I didn't look through your log. I think I fixed it, but you'll need a daily build to use it. What OS are you on? I can hand you a convenient zip.

**Comment by runecalico** [comment]
Windows2012R2 Server with latest patches.

**Comment by da3dsoul** [comment]
Kk I'll build you a zip when I get home

**Comment by da3dsoul** [comment]
http://da3dsoul.shokoanime.com/s/NotAVirus use that, just overwrite the installation.

**Comment by runecalico** [comment]
Awesome, Thanks. I'll do that and let you know what happens :)

**Comment by runecalico** [comment]
Thanks @da3dsoul , this build is able to get past that issue with the Multiple Files (A much needed feature for me). With this I can proceed using Shoko! I had to unzip it on another machine and then copy it over as Windows detected it was a "downloaded" file, and was bitching about not being able to execute/access executables/files.  Once I got past that it worked to show the files and was able to delete them. I consider this issue closed. :)
